src/bootstrap/dylib_util.rs

```diff
@@ -26,3 +26,26 @@ pub fn dylib_path() -> Vec<PathBuf> {
     };
     env::split_paths(&var).collect()
 }
+
+struct LldNoThreads {
+    windows: String,
```

Comment:
```suggestion
    windows: &'static str,
```
This structure is always initialized with string literals.